pytube/cipher.py

```diff
@@ -247,7 +247,7 @@ def splice(arr: List, b: int):
     >>> splice([1, 2, 3, 4], 2)
     [1, 2]
     """
-    return arr[:b] + arr[b * 2 :]
+    return arr[:b]
```

Comment:
can we just review splice function - seems like porting of another language construct for limited marginal utility

Reply:
I don't think we *can* remove the splice function. Pytube basically emulates parts of the javascript in order to generate the video signature.

Reply:
ah good point